### DISCUSSION TITLE
PIL Reader rotate image

### DISCUSSION THREAD

**Discussion by ademyanchuk** [Project-MONAI/MONAI#3220] [Q&A]
Hi, thanks for the great library!

Why `PILReader` in `get_data` method move axes like this `np.moveaxis(np.asarray(i), 0, 1)`
It actually rotates the original image 90 degrees. Did I miss something?

Thanks,
Alexey

**Comment by wyli** [comment]
That's mainly for interoperability. We’d like to ensure that the monai reader and writers with different backends (itk/pil/nibabel) use the same convention.

**Reply by ademyanchuk** [reply]
Thanks. That means, after training predicted segmentations will be also rotated, and one should rotate it back to comply to original image, right?
